LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/IT87XX.cs

```diff
@@ -75,6 +75,7 @@ public IT87XX(Chip chip, ushort address, ushort gpioAddress, byte version, Mothe
         {
             Chip.IT8665E or Chip.IT8625E => new byte[] { 0x15, 0x16, 0x17, 0x1e, 0x1f, 0x92 },
             Chip.IT8792E => new byte[] { 0x15, 0x16, 0x17 },
+            Chip.IT8696E => new byte[] { 0, 0, 0, 0, 0 },
```

Comment:
This broke control support as per https://github.com/Rem0o/FanControl.Releases/issues/3307#issuecomment-3152559770, 

Not quite sure how it made it through but this needs to be removed and the control sensor reading code adjusted if it didn't work without this.